### PR TITLE
Replaced space for extension element name generation for the user (human) task

### PR DESCRIPTION
**Pull request by 4gsim**
#2567 Flowable Modeler - Group names with spaces (e.g., Team Leader, Team Member)

#### Check List:
* Unit tests: YES
* Documentation: NA

### CODE REVIEW

File: modules/flowable-json-converter/src/test/java/org/flowable/editor/language/UsertTaskWithSpacedExtensionIdConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+package org.flowable.editor.language;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.flowable.bpmn.model.BpmnModel;
+import org.flowable.bpmn.model.FlowElement;
+import org.flowable.bpmn.model.UserTask;
+import org.junit.jupiter.api.Test;
+
+public class UsertTaskWithSpacedExtensionIdConverterTest extends AbstractConverterTest {
```
Comment (dbmalkovsky) [inline modules/flowable-json-converter/src/test/java/org/flowable/editor/language/UsertTaskWithSpacedExtensionIdConverterTest.java:24]: I think there is an extra 't` in the class name:  User**t**Task.....

File: modules/flowable-json-converter/src/test/java/org/flowable/editor/language/UsertTaskWithSpacedExtensionIdConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  @Test
+  public void convertJsonToModel() throws Exception {
+    BpmnModel bpmnModel = readJsonFile();
```
Comment (dbmalkovsky) [inline modules/flowable-json-converter/src/test/java/org/flowable/editor/language/UsertTaskWithSpacedExtensionIdConverterTest.java:28]: Project standards use 4 spaces not 2 for indentation.

File: modules/flowable-json-converter/src/test/java/org/flowable/editor/language/UsertTaskWithSpacedExtensionIdConverterTest.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  protected String getResource() {
+    return "test.usertaskmodelwithspacedextensionid.json";
+  }
+
+  private void validateModel(BpmnModel bpmnModel) {
+    FlowElement flowElement = bpmnModel.getMainProcess().getFlowElement("usertask", true);
+    assertThat(flowElement).isInstanceOf(UserTask.class);
+    UserTask userTask = (UserTask) flowElement;
+    assertThat(userTask.getExtensionElements().containsKey("group-info-name-human_resources"))
+        .isTrue();
```
Comment (dbmalkovsky) [inline modules/flowable-json-converter/src/test/java/org/flowable/editor/language/UsertTaskWithSpacedExtensionIdConverterTest.java:49]: You might try simplifying this to (untested):
`assertThat(userTask.getExtensionElements()).containsKey("group-info-name-human_resources");`

File: modules/flowable-json-converter/src/test/java/org/flowable/editor/language/UsertTaskWithSpacedExtensionIdConverterTest.java (hunk 1 above)
Comment (4gsim) [reply modules/flowable-json-converter/src/test/java/org/flowable/editor/language/UsertTaskWithSpacedExtensionIdConverterTest.java]: done

File: modules/flowable-json-converter/src/test/java/org/flowable/editor/language/UsertTaskWithSpacedExtensionIdConverterTest.java (hunk 3 above)
Comment (4gsim) [reply modules/flowable-json-converter/src/test/java/org/flowable/editor/language/UsertTaskWithSpacedExtensionIdConverterTest.java]: Done

**Comment by tijsrademakers** [comment]
Thanks for the PR, but we are not looking for automatic extension element name changes with _. For the issue we need to look for another solution.

**Pull request closed without merging**